hack/generated/pkg/testcommon/kube_test_context_envtest.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package testcommon
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/Azure/k8s-infra/hack/generated/controllers"
+	"github.com/dnaeon/go-vcr/recorder"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/klog/v2/klogr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+)
+
+func createEnvtestContext(perTestContext PerTestContext) (*KubeBaseTestContext, error) {
+	log.Printf("Creating envtest for test %s", perTestContext.TestName)
+
+	environment := envtest.Environment{
+		CRDDirectoryPaths: []string{
+			"../config/crd/bases/valid",
```

Comment:
Should this have a TODO or something that it should be changed... or go away?

Reply:
yes, it should